frontend/app/templates/work_packages.list.new.html

```diff
@@ -0,0 +1,71 @@
+<div cg-busy="vm.loaderPromise" class="work-packages--details-content -create-mode">
+  <div ng-if="vm.workPackage">
+    <h2>{{ ::I18n.t('js.work_packages.create.header') }}</h2>
+
+    <div class="single-attribute subject">
```

Comment:
Not quite understanding the purpose of this `div` container. It got my attention, because I saw it in a new CSS rule in `_work_package.sass`.

* Why is it not to be found in the edit view?
* What is a `single-attribute`? Seems like only the subject and the description are "single attributes".
    * for edit mode only the description is a "single attribute"
* Why do we (again!) need to specify that this is the `subject`?
    * the immediate (and only) DOM child already is a `work-packages--details--subject`
    * the DOM grandchild is an `attribute-subject`
* the class `subject` has no prefix and is likely to collide with other CSS rules